vtds_application_openchami/private/templates/quadlet/prepare_node.sh

```diff
@@ -61,8 +61,18 @@ dnf -y install ${PACKAGES}
 dnf -y install epel-release
 dnf -y install s3cmd
 systemctl enable --now libvirtd
-groupadd rocky
-useradd -g rocky -G libvirt rocky
+if ! grep "^rocky:" /etc/group; then
+    groupadd rocky
+fi
+if ! grep "^rocky:" /etc/passwd; then
+    useradd -g rocky -G libvirt rocky
+fi
 # Remove rocky from /etc/sudoers and then put it back with NOPASSWD access
 sed -i -e '/[[:space:]]*rocky/d' /etc/sudoers
 echo 'rocky ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
+
+# Run OpenCHAMI preparation script as 'rocky'
+cp /root/OpenCHAMI-Prepare.sh ~rocky/OpenCHAMI-Prepare.sh
```

Comment:
I may not be familiar with this Linux distro - is `~rocky` is a name of directory? I don't see where it is created. Tilde `~` expands to home directory of current user (`/root`) only if it is separated from rest of the path with `/`.